Implement a conditional authenticator to check if a sub-flow was executed or not previously in the process (#35668)
keycloak/keycloak: docs/documentation/release_notes/topics/26_1_0.adoc

As OpenShift v3 reached end-of-life a while back, support for identity brokering with OpenShift v3 has been removed from Keycloak.

= New conditional authenticator `Condition - sub-flow executed`

> The `Condition - sub-flow executed` is a new conditional authenticator in {project_name}. The condition checks if a previous sub-flow was executed (or not executed) successfully during the authentication flow execution. More information about conditions in the link:{adminguide_link}#conditions-in-conditional-flows[server admin guide].

Reviewer: ```suggestion
The `Condition - sub-flow executed` is a new conditional authenticator in {project_name}. The condition checks if a previous sub-flow was executed (or not executed) successfully during the authentication flow execution. For more details, see link:{adminguide_link}#conditions-in-conditional-flows[Conditions in conditional flows].
```
